CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ActionPane.java

```diff
@@ -57,12 +80,26 @@ public void handle(ActionEvent t) {
             @Override
             public void handle(ActionEvent t) {
                 try {
-                    importController.processImport();
+                    
+                    List<File> importedFiles = importController.processImport();
+                    String[] filenames = new String[importedFiles.size()];
+                    long noOfRows = 0;
+                    for (int i = 0; i < importedFiles.size(); i++) {
+                        filenames[i] = importedFiles.get(i).getName();
+                        Path path = importedFiles.get(i).toPath();
+                        noOfRows += Files.lines(path).count();
+                    }
+                    displayAlert("Success", "Successfully imported "+ noOfRows
+                            +" rows from the following file(s):\n"+String.join("\n", filenames), true);
+                    
                 } catch (IOException ex) {
                     LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
+                    displayAlert("Import Failed", ex.getLocalizedMessage(), false);
                 } catch (InterruptedException ex) {
                     Thread.currentThread().interrupt();
+                    displayAlert("import Failed", ex.getLocalizedMessage(), false);
                 } catch (PluginException ex) {
```

Comment:
Probably be good to have a log here as well so remove this catch and turn the IO exception catch into a multi-catch with Plugin exceptions